config/database.yml

```diff
@@ -24,7 +24,8 @@ default: &default
 development:
   <<: *default
   database: webpage-versions-db_development
-
+  password: postgres
+  #password: Figaro.env.LOGINROLE_PASSWORD
```

Comment:
I think I would tend towards (2) above because it’s a well known, standard thing.